apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java

```diff
@@ -83,16 +94,22 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
             }
 
             span = ContextManager.createEntrySpan(generateOperationName(requestURL, invocation), contextCarrier);
+            if (DubboPluginConfig.Plugin.Dubbo.COLLECT_PROVIDER_REMOTE_ADDRESS) {
```

Comment:
I prefer you to add this directly, rather than config. This is a light op, should not affect the performance like the arguments.

Reply:
Yes, has remove  please recheck